arbitrum-docs/how-arbitrum-works/02-transaction-lifecycle.mdx

```diff
@@ -1,4 +1,26 @@
-# Overview: The Lifecycle of an Arbitrum Transaction
+---
+title: Transaction Lifecycle
+---
+
+## Sequencing, Followed by Deterministic Execution
+
+This diagram summarizes how transactions are processed in Nitro.
+
+![seq-then-exec](../assets/seq-then-exec.png)
+
+Let's follow a user's transaction through this process.
+
+First, the user creates a transaction, uses their wallet to sign it, and sends it to the Nitro chain's Sequencer. The Sequencer's job, as its name implies, is to take the arriving transactions, put them into an ordered sequence, and publish that sequence.
+
+Once the transactions are sequenced, they are run through the _state transition function_, one by one, in order. The state transition function takes as input the current state of the chain (account balances, contract code, and so on), along with the next transaction. It updates the state and sometimes emits a new Layer 2 block on the Nitro chain.
+
+Because the protocol doesn't trust the Sequencer not to put garbage into its sequence, the state transition function will detect and discard any invalid (e.g., improperly formed) transactions in the sequence. A well-behaved Sequencer will filter out invalid transactions so the state transition function never sees them--and this reduces cost and therefore keeps transactions fees low--but Nitro will still work correctly no matter what the Sequencer puts into its feed. (Transactions in the feed are signed by their senders, so the Sequencer can't create forged transactions.)
+
+The state transition function is deterministic, which means that its behavior depends only on the current state and the contents of the next transaction--and nothing else. Because of this determinism, the result of a transaction T will depend only on the genesis state of the chain, the transactions before T in the sequence, and T itself.
+
+It follows that anyone who knows the transaction sequence can compute the state transition function for themselves--and all honest parties who do this are guaranteed to get identical results. This is the normal way that Nitro nodes operate: get the transaction sequence, and run the state transition function locally. No consensus mechanism is needed for this.
+
+## Overview: The Lifecycle of an Arbitrum Transaction
 
 As an introduction to the various components that compose the Arbitrum protocol, we'll go step-by-step over the phases an Arbitrum transaction goes through, starting with a client creating a signed transaction, to it ultimately being confirmed back on layer 1.
```

Comment:
Two more things about this section:
- The introduction to the section might need some tweaks in the wording. There's a mention to "This overview..." further down.
- The section is full of broken links. Each link should be tested.

Reply:
Addressing phrasing. A few things are a little confusing like referring to phases but this will be resolved in the next iteration. Will test links 👍